### PR TITLE
Issue#267 movie financing ui

### PR DESCRIPTION
**Pull request by pldespaigne**
mimick figma

### CODE REVIEW

File: apps/movie-financing/movie-financing/src/app/explorer/header/header.component.ts [hunk 1]
```diff
@@ -7,6 +7,12 @@ import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class FinancingExplorerHeaderComponent implements OnInit {
+
+  @Input() title = 'Annette';
+  @Input() subtitle = 'Leos Carax';
+  @Input() label = 'logline';
+  @Input() content = 'A stand-up comedian, and his opera singer wife, have a 2 year old daughter with a surprising gift.';
```
[the remark this answers is not in the document]
Comment (GrandSchtroumpf) [reply apps/movie-financing/movie-financing/src/app/explorer/header/header.component.ts]: Or better, use `<ng-content>`

File: apps/movie-financing/movie-financing/src/app/explorer/movie-card/movie-card.component.scss [hunk 2]
```diff
@@ -1,3 +1,123 @@
...
+  margin-bottom: 30px;
+  width: 60%;
+  h3 {
+    font-weight: 300;
+    text-transform: uppercase;
+    letter-spacing: 7px;
+    color: #c6c6c6;
+  }
+}
+progress {
```
Comment (GrandSchtroumpf) [inline apps/movie-financing/movie-financing/src/app/explorer/movie-card/movie-card.component.scss]: There is an Angular Material Component for that. Use it.

File: apps/movie-financing/movie-financing/src/app/explorer/profile/profile.component.html [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+          <span fxFlex>
+            <label for="society">Society</label>
+            <input type="text" name="society" placeholder="Input text"/>
+          </span>
+          <span fxFlex><!--spacer--></span>
+        </div>
+      </div>
+    </section>
+
+    <section class="margin-t-60">
```
Comment (GrandSchtroumpf) [inline apps/movie-financing/movie-financing/src/app/explorer/profile/profile.component.html]: don't use `margin-t-60` kind of class. You do that when you build a component library. Just apply the style in the scss

File: apps/movie-financing/movie-financing/src/app/explorer/profile/profile.component.scss [hunk 4]
```diff
@@ -0,0 +1,107 @@
+@import url('https://fonts.googleapis.com/css?family=Montserrat:200,300,400,500,600,700,800,900');
+
+// LAYOUT
+
+main {
+  max-height: none !important;
```
Comment (GrandSchtroumpf) [inline apps/movie-financing/movie-financing/src/app/explorer/profile/profile.component.scss]: Don't use `!important`.

File: apps/movie-financing/movie-financing/src/app/explorer/profile/profile.component.html [hunk 5]
```diff
@@ -0,0 +1,90 @@
+<financing-explorer-header
```
Comment (GrandSchtroumpf) [inline apps/movie-financing/movie-financing/src/app/explorer/profile/profile.component.html]: If you're using the header and footer several time, maybe you can think about using a routing with children

File: apps/movie-financing/movie-financing/src/app/explorer/ui/range-slider/range-slider.component.scss [hunk 6]
```diff
@@ -0,0 +1,50 @@
+input {
+  border: 1px solid rgba(25, 25, 25, 0.32);
+  height: 43px;
+  width: 100%;
+}
+input::placeholder {
+  font-size: 16px;
+}
+input[type=range] {
```
Comment (GrandSchtroumpf) [inline apps/movie-financing/movie-financing/src/app/explorer/ui/range-slider/range-slider.component.scss]: Use `mat-input`

File: apps/movie-financing/movie-financing/src/app/explorer/ui/range-slider/range-slider.component.ts [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+})
+export class FinancingRangeSliderComponent implements OnInit {
+  @Input() min = 0;
+  @Input() max = 100;
+  @Input() step = 1;
+  @Input() value: number;
+  @Input() placeholder: string;
+  @Input() valueOffset = 20;
+  @Input() unit: string;
+  @Input() unitSide = 'none';
```
Comment (GrandSchtroumpf) [inline apps/movie-financing/movie-financing/src/app/explorer/ui/range-slider/range-slider.component.ts]: There is a angular material component for that. Use it.

File: libs/shared-assets/apps/movie-financing/range-thumb.svg [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+<feFlood flood-opacity="0" result="BackgroundImageFix"/>
+<feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/>
+<feOffset dy="2"/>
+<feGaussianBlur stdDeviation="7"/>
+<feColorMatrix type="matrix" values="0 0 0 0 0.196078 0 0 0 0 0.196078 0 0 0 0 0.196078 0 0 0 0.392346 0"/>
+<feBlend mode="normal" in2="BackgroundImageFix" result="effect1_dropShadow"/>
+<feBlend mode="normal" in="SourceGraphic" in2="effect1_dropShadow" result="shape"/>
+</filter>
+</defs>
+</svg>
```
Comment (GrandSchtroumpf) [inline libs/shared-assets/apps/movie-financing/range-thumb.svg]: Use indentation.

File: apps/movie-financing/movie-financing/src/app/explorer/header/header.component.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (pldespaigne) [reply apps/movie-financing/movie-financing/src/app/explorer/header/header.component.ts]: ✔️

File: apps/movie-financing/movie-financing/src/app/explorer/movie-card/movie-card.component.html [hunk 9]
```diff
@@ -1,7 +1,43 @@
...
-  <img mat-card-image src="https://material.angular.io/assets/img/examples/shiba2.jpg" alt="Photo of a Shiba Inu">
+<mat-card>
+  <div class="poster-container">
+    <img class="movie-poster" src="https://fakeimg.pl/657x251/" alt="Movie Poster">
+    <div class="movie-info">
+      <div class="subtitle">Presitge Directors</div>
+      <div class="title">
+        <div class="stroke">Presitge Directors</div>
+        <div class="director-name">Director Name</div>
+        <div class="movie-title">Movie Name soooooo Long</div>
```
[the remark this answers is not in the document]
Comment (pldespaigne) [reply apps/movie-financing/movie-financing/src/app/explorer/movie-card/movie-card.component.html]: ✔️

File: apps/movie-financing/movie-financing/src/app/explorer/movie-card/movie-card.component.scss (hunk 2 above)
Comment (pldespaigne) [reply apps/movie-financing/movie-financing/src/app/explorer/movie-card/movie-card.component.scss]: ✔️

File: apps/movie-financing/movie-financing/src/app/explorer/movie-card/movie-card.component.scss [hunk 10]
```diff
@@ -1,3 +1,123 @@
...
+      background-color: white;
+      width: 40%;
+      height: 32px;
+      padding-top: 10px;
+      padding-left: 20px;
+      text-transform: uppercase;
+      letter-spacing: 8px;
+    }
+  }
+  .title {
```
[the remark this answers is not in the document]
Comment (pldespaigne) [reply apps/movie-financing/movie-financing/src/app/explorer/movie-card/movie-card.component.scss]: ✔️

File: apps/movie-financing/movie-financing/src/app/explorer/profile/profile.component.html (hunk 3 above)
Comment (pldespaigne) [reply apps/movie-financing/movie-financing/src/app/explorer/profile/profile.component.html]: ✔️

File: apps/movie-financing/movie-financing/src/app/explorer/profile/profile.component.scss (hunk 4 above)
Comment (pldespaigne) [reply apps/movie-financing/movie-financing/src/app/explorer/profile/profile.component.scss]: ✔️

File: libs/shared-assets/apps/movie-financing/range-thumb.svg (hunk 8 above)
Comment (pldespaigne) [reply libs/shared-assets/apps/movie-financing/range-thumb.svg]: ✔️

File: apps/movie-financing/movie-financing/src/app/explorer/ui/range-slider/range-slider.component.scss (hunk 6 above)
Comment (pldespaigne) [reply apps/movie-financing/movie-financing/src/app/explorer/ui/range-slider/range-slider.component.scss]: ✔️

File: apps/movie-financing/movie-financing/src/app/explorer/ui/range-slider/range-slider.component.ts (hunk 7 above)
Comment (pldespaigne) [reply apps/movie-financing/movie-financing/src/app/explorer/ui/range-slider/range-slider.component.ts]: ✔️

File: apps/movie-financing/movie-financing/src/app/explorer/profile/profile.component.html (hunk 5 above)
Comment (pldespaigne) [reply apps/movie-financing/movie-financing/src/app/explorer/profile/profile.component.html]: This need more sync beetween @lsenta @phpgeekfr and me, for the moment the movie-financing architecture is a bit weird, so I will treat that in a separate issue #304

**Pull request merged**